src/jdk.sctp/share/classes/com/sun/nio/sctp/SctpStandardSocketOptions.java

```diff
@@ -34,22 +34,22 @@
  *
  * @since 1.7
  */
-public class SctpStandardSocketOptions {
+public final class SctpStandardSocketOptions {
     private SctpStandardSocketOptions() {}
     /**
      * Enables or disables message fragmentation.
      *
      * <P> The value of this socket option is a {@code Boolean} that represents
      * whether the option is enabled or disabled. If enabled no SCTP message
-     * fragmentation will be performed. Instead if a message being sent
+     * fragmentation will be performed. Instead, if a message being sent
      * exceeds the current PMTU size, the message will NOT be sent and
      * an error will be indicated to the user.
      *
      * <P> It is implementation specific whether or not this option is
      * supported.
      */
     public static final SctpSocketOption<Boolean> SCTP_DISABLE_FRAGMENTS = new
-        SctpStdSocketOption<Boolean>("SCTP_DISABLE_FRAGMENTS", Boolean.class,
+        SctpStdSocketOption<>("SCTP_DISABLE_FRAGMENTS", Boolean.class,
         sun.nio.ch.sctp.SctpStdSocketOption.SCTP_DISABLE_FRAGMENTS);
```

Comment:
I guess you don't need fully qualified import for `sun.nio.ch.sctp.SctpStdSocketOption` as it is already imported in this class